docs/configure

```diff
@@ -478,10 +480,11 @@ ditch_maxmsgsize:
 		and domain = '${quote_mysql:$domain}' \
 		and users.maxmsgsize > 0 \
 		and users.domain_id=domains.domain_id}{${value}K}fail} Kb.
-  local_part_suffix = -*
-  local_part_suffix_optional
+#  local_part_suffix = -*
+#  local_part_suffix_optional
```

Comment:
Please explain. :)

I just had an idea: would it be possible to do like Debian does and define a macro/variable/whatever at the top of `configure` which could be easily changed, and which would switch these options on or off? Something like this:

```
VEXIM_LOCALPART_SUFFIX = -*

...

.ifdef VEXIM_LOCALPART_SUFFIX
local_part_suffix = VEXIM_LOCALPART_SUFFIX
local_part_suffix_optional
.endif
```